src/DataProvider/OperationDataProviderTrait.php

```diff
@@ -106,7 +106,7 @@ private function extractIdentifiers(array $parameters, array $attributes)
                         throw new InvalidIdentifierException(sprintf('Expected %d identifiers, got %d', $identifiersNumber, $currentIdentifiersNumber));
                     }
 
-                    return $identifiers;
```

Comment:
This is the expected behavior now, the composite identifier parsing is done here: https://github.com/api-platform/core/blob/master/src/DataProvider/OperationDataProviderTrait.php#L102-L110 

I haven't skipped any functional test (behat), it seems weird to me that the behavior changed as it shouldn't. May you give me a reproducible case?

Reply:
The code you mentionned is actually the one that I modified @soyuka, you might have read to fast

parsing is still ok but if your composite identifer is composed of an integer property and a string property, you still need to denormalize them.


Example would be
```php
@ApiResource
class Foo
{
    public int $identifer1;
    public string $identifier2;
}
```

when requesting `/foos/identifer1=1;identifier2=test`, without the patch you would get an identifier array like this
```php
[
    "identifer1" => "1",
    "identifer2" => "test",
]
```

where I expect 
```php
[
    "identifer1" => 1,
    "identifer2" => "test",
]
```

Reply:
that's what was tested with integer and datetime line 57  and 58 of the test mentioned in the PR description

Reply:
you can ping me on slack if you have time